deploy/ocp/manifests/0.13.0/0000_50_olm_07-olm-operator.deployment.yaml

```diff
@@ -64,6 +64,9 @@ spec:
           - name: OPERATOR_NAME
             value: olm-operator
           resources:
+            limits:
```

Comment:
Since the `0.13.0` release was already cut, we don't want to change its manifests. These changes will be included when we cut the next release.

Please exclude all changes to the `deploy/ocp/manifests/0.13.0` directory.

Reply:
Ok good call @njhale I was wondering about that